ratelimit_test.go

```diff
@@ -7,10 +7,18 @@ import (
 
 	"go.uber.org/atomic"
 
-	"github.com/benbjohnson/clock"
 	"github.com/stretchr/testify/assert"
 )
 
+func (tt *testTime) Now() time.Time {
```

Comment:
why did you keep it here as opposed to the other file with other methods?

Reply:
I thought that we could keep in ratelimit_mock_time_test.go things that were adopted from golang/time and here we just implement our own clock interface.

Reply:
Reasonable idea.

Separately, I'm surprised we don't get a linter failures about missing docstrings. Some checks are missing, but that's not related to your change.